runtime/compiler/module.xml

```diff
@@ -23,6 +23,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 <module>
 	<artifact type="target" name="j9jitlauncher">
 		<include-if condition="spec.flags.interp_nativeSupport"/>
+		<exclude-if condition="spec.linux_riscv.*" />
```

Comment:
Please revert changes such as this which are now unnecessary (in all module.xml files).

Reply:
Removed all exclude conditions specific to RISC-V and updated the copyright date every where.